app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -348,23 +372,127 @@ class AppointmentRepositoryAndroidTest {
   @Test
   fun when_removing_appointment_with_reason_as_patient_already_visited_then_appointment_should_be_marked_as_visited() {
     val patientId = UUID.randomUUID()
-    val appointmentDate = LocalDate.now()
-    repository.schedule(patientId, appointmentDate).blockingAwait()
+    val appointmentDate = LocalDate.now(clock)
+    appointmentRepository.schedule(patientId, appointmentDate).blockingAwait()
 
-    val appointments = repository.recordsWithSyncStatus(SyncStatus.PENDING).blockingGet()
+    val appointments = appointmentRepository.recordsWithSyncStatus(SyncStatus.PENDING).blockingGet()
     assertThat(appointments).hasSize(1)
     assertThat(appointments.first().status).isEqualTo(Appointment.Status.SCHEDULED)
 
     val uuid = appointments[0].uuid
-    repository.setSyncStatus(listOf(uuid), SyncStatus.DONE).blockingGet()
-    repository.markAsVisited(uuid).blockingGet()
+    appointmentRepository.setSyncStatus(listOf(uuid), SyncStatus.DONE).blockingGet()
+    appointmentRepository.markAsVisited(uuid).blockingGet()
 
-    val updatedList = repository.recordsWithSyncStatus(SyncStatus.PENDING).blockingGet()
+    val updatedList = appointmentRepository.recordsWithSyncStatus(SyncStatus.PENDING).blockingGet()
     assertThat(updatedList).hasSize(1)
     updatedList[0].apply {
       assertThat(this.uuid).isEqualTo(uuid)
       assertThat(this.cancelReason).isNull()
       assertThat(this.status).isEqualTo(Appointment.Status.VISITED)
     }
   }
+
+  @Test
+  @Suppress("ASSIGNING_SINGLE_ELEMENT_TO_VARARG_IN_NAMED_FORM_FUNCTION")
```

Comment:
I just realized that this is a deprecation warning, and the ability to provide single arguments to named vararg parameters has been removed in Kotlin 1.3 (which is stable and released now). Upgrading our Kotlin version to 1.3 will break these tests.

Reply:
Oh no. The snytax for providing vararg parameters is not nice :( 

I'll update this.

Reply:
Fixed

Reply:
> Oh no. The snytax for providing vararg parameters is not nice :(
> 
> I'll update this.

Yeah. Maybe we should avoid varargs altogether and just use lists everywhere.